raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -85,29 +85,33 @@ contract TokenNetwork is Utils {
      */
 
     event ChannelOpened(
-        bytes32 channel_identifier,
-        address participant1,
-        address participant2,
+        bytes32 indexed channel_identifier,
+        address indexed participant1,
+        address indexed participant2,
         uint256 settle_timeout
     );
 
-    event ChannelNewDeposit(bytes32 channel_identifier, address participant, uint256 deposit);
+    event ChannelNewDeposit(bytes32 indexed channel_identifier, address indexed participant, uint256
+    deposit);
 
     // withdrawn_amount is the total amount withdrawn by the participant from the channel
-    event ChannelWithdraw(bytes32 channel_identifier, address participant, uint256 withdrawn_amount);
+    event ChannelWithdraw(bytes32 indexed channel_identifier, address indexed participant, uint256
+    withdrawn_amount);
```

Comment:
How didn't I see this code style change? Please don't use this and stick to our code style https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md ...